pymatgen/analysis/quasirrho.py

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) Pymatgen Development Team.
+# Distributed under the terms of the MIT License.
+
+
+"""
+A module to calculate free energies using the Quasi-Rigid Rotor Harmonic
+ Oscillator approximation. Modified from a script by Steven Wheeler.
+See: Grimme, S. Chem. Eur. J. 2012, 18, 9955
+"""
+
+__author__ = "Alex Epstein"
+__copyright__ = "Copyright 2020, The Materials Project"
+__version__ = "0.1"
+__maintainer__ = "Alex Epstein"
+__email__ = "aepstein@lbl.gov"
+__date__ = "January 5, 2021"
+__credits__ = "Steven Wheeler, Trevor Seguin, Evan Spotte-Smith"
+
+from math import isclose
+from typing import Union
+
+import numpy as np
+import scipy.constants as const
+
+from pymatgen.core.units import amu_to_kg
+from pymatgen.core.units import kb as kb_ev
+from pymatgen.io.gaussian import GaussianOutput
+from pymatgen.io.qchem.outputs import QCOutput
+
+# Define useful constants
+kb = kb_ev * const.eV  # pymatgen kb from ev/K to J/K
+c = const.speed_of_light * 100  # cm/s
+h = const.h  # Planck's constant J.s
+R = const.R / const.calorie  # Ideal gas constant cal/mol
+
+# Define useful conversion factors
+kcal2hartree = 0.0015936  # kcal/mol to hartree/mol
+
+
+def get_avg_mom_inertia(mol):
+    """
+    Caclulate the average moment of inertia of a molecule
+    :param mol: Molecule
+    :return: average moment of inertia, eigenvalues of inertia tensor
+    """
+    centered_mol = mol.get_centered_molecule()
+    inertia_tensor = np.zeros((3, 3))
+    for site in centered_mol:
+        c = site.coords
+        wt = site.specie.atomic_mass
+        for i in range(3):
+            inertia_tensor[i, i] += wt * (c[(i + 1) % 3] ** 2 + c[(i + 2) % 3] ** 2)
+        for i, j in [(0, 1), (1, 2), (0, 2)]:
+            inertia_tensor[i, j] += -wt * c[i] * c[j]
+            inertia_tensor[j, i] += -wt * c[j] * c[i]
+
+    inertia_eigenvals = np.multiply(np.linalg.eig(inertia_tensor)[0], amu_to_kg * 1e-20).tolist()  # amuangs^2 to kg m^2
+
+    iav = np.average(inertia_eigenvals)
+
+    return iav, inertia_eigenvals
+
+
+class QuasiRRHO:
+    """
+    Class to calculate thermochemistry using Grimme's Quasi-RRHO approximation.
```

Comment:
May add the citation to the paper in this docstring so that it's visible in, e.g., Jupyter Notebook and documentation pages?

Reply:
Thanks for adding to the docstring! Since you opened this PR, we have also added a nice system for citations in `pymatgen` based on [duecredit](https://github.com/duecredit/duecredit). Would you mind adding a @due.dcite decorator to this class? ([example](https://github.com/materialsproject/pymatgen/blob/91dbe6ee9ed01d781a9388bf147648e20c6d58e0/pymatgen/core/lattice.py#L1168-L1172))